Task 1: Заполнить все данные по станциям метро.

abishkam opened the pull request:
- Добавил все станции

abishkam committed:
Task 1: Заполнить все данные по станциям метро.
- Добавил все станции

vicria commented on subte/subte-microservice/src/main/resources/db/migration/common/V01_00_00_006__insert_remaining_stations.sql:
```diff
@@ -0,0 +1,48 @@
+INSERT INTO subte_bot.subte_station (id, line, name) VALUES (gen_random_uuid()::text, 'C🔵', 'San Juan');
+INSERT INTO subte_bot.subte_station (id, line, name) VALUES (gen_random_uuid()::text, 'C🔵', 'Constitución');
+
+INSERT INTO subte_bot.subte_station (id, line, name) VALUES (gen_random_uuid()::text, 'A🔵', 'Plaza de Mayo');
```
я видимо непонятно в личке написала. давай заменим А на 🌐 эмодзи

vicria commented on subte/subte-microservice/src/main/resources/db/migration/common/V01_00_00_007__insert_connection_blueC.sql:
```diff
@@ -0,0 +1,29 @@
+SELECT insert_connection('Retiro', 'C🔵', 'General San Martín', 'C🔵', 'Constitución', 'C🔵', 3);
```
ты добавил связи между ветками, значит нужно размножить тест, где мы ходим о начальной станции ветки до конечной и обратно

abishkam committed:
Task 1: Заполнить все данные по станциям метро.
- Исправил A🔵 на A🌐
- Добавил станции в тесты

abishkam committed:
Task 1: Заполнить все данные по станциям метро.
- Добавил переходы от старых линий к новым
- желательно перед тем, как запустить 10ю миграцию - поменять её название

vicria reviewed (approved):
всё отлично)
